zip/zip.go

```diff
@@ -50,7 +49,7 @@ func Zip(dirPath string, insidePath string) ([]string, []byte, error) {
 		return nil
 	})
 	if err != nil {
-		return nil, nil, fmt.Errorf("walk directory %s", dirPath)
+		return nil, nil, errors.Wrap(err, fmt.Sprintf("walk directory %s", dirPath))
```

Comment:
Won't the `dirPath` already be present in the wrapped `err`?

Reply:
It depends on an error returned. `dirPath` represents root dir passed for zipping. It will be present for some errors, but for most of them, the current path (where an error occurred) would be present.